src/plugins/collapsibleColumns/collapsibleColumns.js

```diff
@@ -287,6 +303,25 @@ class CollapsibleColumns extends BasePlugin {
     return divEl;
   }
 
+  /**
+   * Generates the list of collapsable coords.
+   *
+   * @private
+   * @param {Number} column Column index.
+   * @param {HTMLElement} TH TH Element.
+   */
+  generateCollapsableCoordsList(column, TH) {
+    const TR = TH.parentNode;
+    const THEAD = TR.parentNode;
+    const row = ((-1) * THEAD.childNodes.length) + Array.prototype.indexOf.call(THEAD.childNodes, TR);
```

Comment:
What about using `Array.from` instead of messing with Array prototype?
```suggestion
    const row = ((-1) * THEAD.childNodes.length) + Array.from(THEAD.childNodes).indexOf(TR);
```